src/lib/utility.js

```diff
@@ -124,6 +124,57 @@ module.exports = (requires) => {
   utilities.filter = (str) => {
     return str.replace('%prefix', config.prefix);
   };
+
+  //for finding the command object for either the keyword or an alias 
+  utilities.getCommand = function(keyword) 
```

Comment:
For consistency, use an arrow function here and have the bracket open on the same line
```suggestion
  utilities.getCommand = (keyword) => {
```